### PR TITLE
Update docker docs

### PR DESCRIPTION
**Pull request by agjohnson**
Small edits

### CODE REVIEW

File: docs/development/buildenvironments.rst [hunk 1]
```diff
@@ -16,20 +16,19 @@ any development on the Docker build system, you will need to set up `Docker`_ on
...
+images are found on our `Docker Hub repository`_, the source comes from our
+`container image repo`_.
 
-To get started, create the image using the `docker` command line tool. You can
-name the image whatever you like here, ``rtfd-build`` is the default name, but
-can be configured in your settings -- see `Configuration`_::
+To get started, pull one of the build images down::
 
-    docker build -t rtfd-build base/
+    docker pull readthedocs/build:latest
```
Comment (humitos) [inline docs/development/buildenvironments.rst]: What is the build that it's used in production? I suppose it's the `2.0`, right? If so, I think it's a good idea to mention that `latest` is good for development but to reproduce some someone else's issues it's useful to setup the `2.0`. On the other hand, if `2.0` will be in production for some more time, it's a good idea to test with `2.0`.

Anyway, I'm not sure if this is the proper place to clarify this, maybe another section to this page could be added, or in testing. Not sure.

Comment (agjohnson) [inline docs/development/buildenvironments.rst]: Correct, this was me being lazy. Putting `2.0` there is likely to go stale, where `latest` wouldn't. Plus, I would like to point folks to test there, but that's perhaps a different issue.

I think you're right, I'm going to push up a fancier version.

**Comment by agjohnson** [comment]
Updated this with a fancy version of keeping settings current.

File: docs/conf.py [hunk 2]
```diff
@@ -6,6 +6,7 @@
 from recommonmark.parser import CommonMarkParser
 
 sys.path.insert(0, os.path.abspath('..'))
+sys.path.append(os.path.dirname(__file__))
```
Comment (ericholscher) [inline docs/conf.py]: Should this replace the above insert?

Comment (agjohnson) [inline docs/conf.py]: The first insert is the root path, ie - `docs/../`, for our code, the second insert is the `docs/` path.

**Comment by agjohnson** [comment]
Rebased this to fix the tests.

File: docs/conf.py (hunk 2 above)
Comment (ericholscher) [inline docs/conf.py]: :+1:

**Comment by ericholscher** [comment]
Actually -- looking at this, any reason you didn't put it in the existing `_ext`?>

**Comment by agjohnson** [comment]
Oh i didn't realize that was a thing. I can open a new PR

**Pull request merged**